github/src/utils/actionUtils.ts

```diff
@@ -0,0 +1,110 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { TreeItemIconPath, nonNullProp } from "@microsoft/vscode-azext-utils";
+import * as dayjs from "dayjs";
+// eslint-disable-next-line import/no-internal-modules
+import * as relativeTime from 'dayjs/plugin/relativeTime';
+import { ThemeColor, ThemeIcon, l10n } from "vscode";
+import { ActionWorkflowRuns } from "../wrappers/getActions";
+import { Job, JobStep } from "../wrappers/getJobs";
+
+dayjs.extend(relativeTime);
+
+// https://developer.github.com/v3/checks/runs/#parameters
+export enum Conclusion {
+    Success = 'success',
+    Failure = 'failure',
+    Skipped = 'skipped',
+    Cancelled = 'cancelled'
+}
+
+export enum Status {
+    Queued = 'queued',
+    InProgress = 'in_progress',
+    Completed = 'completed'
+}
+
+// Description
+export function getJobBasedDescription(data: Job | JobStep): string {
+    if (data.conclusion !== null) {
+        return l10n.t('{0} {1}', convertConclusionToVerb(<Conclusion>nonNullProp(data, 'conclusion')), dayjs(data.completed_at).fromNow());
```

Comment:
convertConclusionToVerb is already localized below. I'm not sure how embedded localized strings work.

Reply:
Yeah, this was a product of the original I pulled it from, but I kind of liked how it accidentally made the returned string a lot more readable.  Would you prefer I change this?

Reply:
Yeah, I say we just change it if it doesn't make any sense.